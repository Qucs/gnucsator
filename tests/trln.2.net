# Qucs

#Pac:P1 _net0 gnd Num="1" Z="50 Ohm" P="0" f="1 GHz" Temp="26.85"

Pac:P2 _net0 gnd Num="1" Z="50 Ohm" P="1" f="1 GHz" Temp="26.85"
TLIN:Line1 _net0 _net1 Z="50 Ohm" L="100 mm" Alpha="0 dB" Temp="26.85"
TLIN:Line2 _net1 _net2 Z="50 Ohm" L="100 mm" Alpha="0 dB" Temp="26.85"

R:R1 gnd _net2 R="100 Ohm"
R:R2 gnd _net1 R="100 Ohm"

.AC:SP1 Type="log" Start="1e5" Stop="1e9" Points="6"
