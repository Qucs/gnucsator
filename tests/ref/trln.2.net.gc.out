<Qucs Dataset>
default plugins: ignored
#Pac:P1 _net0 gnd Num="1" Z="50 Ohm" P="0" f="1 GHz" Temp="26.85" 
^ ? #Pac: no match
Gnucap   System status
iterations: op=3, dc=0, tran=0, fourier=0, total=9
transient timesteps: accepted=0, rejected=0, total=0
transient timesteps: accepted=0, rejected=0, total=0
nodes: user=3, subckt=1, model=0, total=4
dctran density=62.5%, ac density=75.0%

#Freq                      vr(_net0)                 vr(_net1)                 vr(_net2)                 vi(_net0)                 vi(_net1)                 vi(_net2)                
 100.E+3                   9.9999995227723932345     9.9999994678652921464     9.9999992756905093926    -785.94173410547805147E-6 -0.0028817868219399226603 -0.0039297093356495050642 
 630.95734448019360485E+3  9.9999811438336578817     9.9999789579113418369     9.9999713072864420126    -0.0049589217683148429927 -0.018182826600730944477  -0.024794771429129851547  
 3.981071705534976246E+6   9.9992495057957544446     9.9991624330456030378     9.9988578419867426561    -0.031279800247055034034  -0.11472109825937175653   -0.15643984136536532858   
 25.118864315095832395E+6  9.9702356229601534209     9.9666906673587458698     9.9545422048877902909    -0.195135603626759071     -0.72264080562780739214   -0.98591537837240714736   
 158.48931924611164845E+6  8.9873242563893835921     8.7284915727405412155     8.2108090111857041648    -0.71279738798289580259   -4.2650044731336835823    -5.927782255545830381     
 1.0000000000000024424E+9  9.7711147490170731089    -1.8119863672340681404    -4.8985705851177776537    -3.5662045504266357554    -7.0634712706997824227     9.8634516486926102629    
